buildroot/share/PlatformIO/scripts/alfawise_Ux0.py

```diff
@@ -5,7 +5,7 @@
     if define[0] == "VECT_TAB_ADDR":
         env['CPPDEFINES'].remove(define)
 env['CPPDEFINES'].append(("VECT_TAB_ADDR", "0x08010000"))
-env.Replace(LDSCRIPT_PATH="buildroot/share/PlatformIO/ldscripts/alfawise_Ux0.ld")
+env.Replace(LDSCRIPT_PATH="buildroot/share/PlatformIO/ldscripts/longer_Ux0.ld")
```

Comment:
Ux0 was for alfawise "U20 U20+ U30", LK or STM would be a good replacement of the suffix...

Reply:
Makes sense. Change applied.